MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -20,7 +20,7 @@ class RouteManeuverViewController: UIViewController {
         didSet {
             if isViewLoaded {
                 roadCode = step?.codes?.first ?? step?.destinationCodes?.first ?? step?.destinations?.first
-                updateStreetNameForStep()
+                updateStreetNameForStep(currentLeg: nil)
```

Comment:
If I’m not mistaken, this would set up a race condition in which the label initially displays the Directions API–provided name (e.g., “Race Street”) until the first location update, at which point it would display the developer-provided name (“Starbucks” or “102 Race Street”). https://github.com/mapbox/mapbox-navigation-ios/pull/419#discussion_r129280021 would solve this issue.